examples/mobile/UIComponents/js/app.js

```diff
@@ -36,12 +38,21 @@
 		dropDownMenuWidget.open();
 	}
 
-	document.addEventListener("pagebeforeshow", function () {
+	document.addEventListener("pageshow", function () {
+		themeChanger = document.querySelector(".ui-page-active select[data-native-menu='false']"),
```

Comment:
`data-native-menu="false"` this is not enough specific selector. You can add some specific class to this `select` element.
By the way, I think the attribute 'data-native-menu="false"' is not necessary - this value is set by default.

Reply:
default data-native-menu is true not false, see
https://github.com/Samsung/TAU/blob/tau_1.2/src/js/profile/mobile/widget/DropdownMenu.js
```
self.options = {
						nativeMenu: true,

```

changed selector to searchfor additional class